www/grid_view/.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.{coffee,less,jade,json}]
```

Comment:
Maybe I could move this to the root of the repo then ?

The default indentation rules for coffee seems to be 2 spaces.